src/fhirtypes/StructureDefinition.ts

```diff
@@ -376,6 +379,11 @@ type PathPart = {
   brackets?: string[];
 };
 
+export type Narrative = {
```

Comment:
I think we should probably pull this into one of the data types files.  FHIR lists this in a weird section on the data types page, so maybe we should just put it into the `dataTypes.ts`?